src/jobs/pages/job-offers-admin.component.vue

```diff
@@ -62,24 +65,49 @@ import Dialog from "primevue/dialog";
               ><i :class="PrimeIcons.SEARCH" />
               <InputText
                 v-model="filters['global'].value"
-                placeholder="Search..." />
+                placeholder="🔍 Search..." />
```

Comment:
There's a way to add icons at the start of the input with [PrimeVue](https://primefaces.org/primevue/inputtext).

It would be something like this:

```vue
<span class="p-input-icon-left">
  <i :class="PrimeIcons.SEARCH" />
  <InputText v-model="filters['global'].value" placeholder="Search..." />
</span>
```